cargo/jam/summarize_test.go

```diff
@@ -431,4 +431,20 @@ version = "3.4.5"
 			}`))
 		})
 	})
+
+	context.Focus("failure cases", func() {
```

Comment:
Can you remove this focus?

Reply:
This is why I shouldn't write code without a pair. Removed.

Reply:
LOL it's easy to miss